ios/RnExecutorch/models/object_detection/SSDLiteLargeModel.mm

```diff
@@ -0,0 +1,77 @@
+#include "SSDLiteLargeModel.hpp"
+#include "../../utils/ObjectDetectionUtils.hpp"
+#include "ImageProcessor.h"
+#include <vector>
+
+inline float constexpr iouThreshold = 0.55;
+inline float constexpr detectionThreshold = 0.7;
+inline int constexpr inputWidth = 320;
+inline int constexpr inputHeight = 320;
```

Comment:
Fun fact, starting from c++17 every constexpr variable is also always implicitly inline. Also they can be defined in header files, that's what inline means (it has nothing to do with inlining the variable by the compilator)
```suggestion
float constexpr iouThreshold = 0.55;
float constexpr detectionThreshold = 0.7;
int constexpr inputWidth = 320;
int constexpr inputHeight = 320;
```